src/fileTransformers/mysqlSchemaTypeTransformer.ts

```diff
@@ -0,0 +1,11 @@
+import { FileTransformer } from '~/src/fileTransformer/fileTransformer'
+
+export const mysqlSchemaTypeTransformer: FileTransformer = (params) => {
+  const { file, parameters } = params
+
+  if (parameters.datasourceProvider !== 'mysql') {
+    return file.content
+  }
+
+  return file.content.replace(/(String\?|String)/g, `$1 @db.Text`)
```

Comment:
```suggestion
  return file.content.replace(/ (String\??) /g, ` $1 @db.Text`)
```

Hm, should work. I am guessing the issue before was the `$1` not being spaced?

I would prefer to keep the leading and trailing spaces to avoid `String` colliding with other parts of the schema (its a pretty common word, prefix/suffix give us a bit more buffer for free).

Reply:
Minor point though, overall LGTM!

Reply:
You change doesn't work for `String?` , only for `String`. 
![image](https://user-images.githubusercontent.com/11679005/183467620-3a3c6b66-d084-434c-aad7-8a919feec9b1.png)

Reply:
But it does work without the last whitespace:
```suggestion
  return file.content.replace(/ (String\??)/g, ` $1 @db.Text`)
```

Reply:
But it does work without the last whitespace:
```suggestion
  return file.content.replace(/ (String\??)/g, ` $1 @db.Text`)
```

Reply:
But it does work without the last whitespace:
```suggestion
  return file.content.replace(/ (String\??)/g, ` $1 @db.Text`)
```

Reply:
But it does work without the last whitespace:

```suggestion
  return file.content.replace(/ (String\??)/g, ` $1 @db.Text`)
```

Reply:
That's surprising since there's definitely a space trailing `?`, must be missing something (like its different kind of white space character than space, etc.... who knows).


Anyways seems that you found a working near-same version!

Reply:
Maybe there is no space after at all, if you take a look at the `name String?` there is nothing after.
I kind of think that the failing test is expected. What do you think?